src/types/global.types.ts

```diff
@@ -0,0 +1,89 @@
+interface Bug {
```

Comment:
shouldn't the file be `global.d.ts`? I haven't seen `.types.ts` before

Reply:
Oh you're right, I didn't realize that's the proper way to declare compile-time constants(?) will update!

Reply:
I wonder if I'm overpopulating the global types now that I think about it. 🤔 Based on [this](https://basarat.gitbook.io/typescript/project/modules/globals), wonder if it's best to use file based modules for most of these types? But I guess we don't have too many to work with right now.

Reply:
Idk if there's really an issue with "polluting" the global namespace. And the goal for this issue is to share types across modules without needing imports. So if we didn't put them in a global declarations file, seems like we could just leave them where they are.

Reply:
let's ask someone with more TS experience @brennie

Reply:
I find types being automatically imported from global.d.ts surprising tbh.

global.d.ts is, imo, for when you need to extend things already in the global namespace (HTML*Element, etc) to add custom properties, etc.

I would advise against putting all these types here.